public/app/features/alerting/unified/components/mute-timings/MuteTimingTimeRange.tsx

```diff
@@ -23,18 +27,6 @@ export const MuteTimingTimeRange = ({ intervalIndex }: Props) => {
     name: `time_intervals.${intervalIndex}.times`,
   });
 
-  const validateTime = (timeString: string) => {
-    if (!timeString) {
-      return true;
-    }
-    const [hour, minutes] = timeString.split(':').map((x) => parseInt(x, 10));
```

Comment:
Removed all of this in favor of a regular expression I've copied from the Alertmanager source. Previous implementation contained small bugs and would allow invalid syntax.